base/applications/fontview/lang/bg-BG.rc

```diff
@@ -10,6 +10,12 @@ BEGIN
     IDS_ERROR_NOMEM "Няма достатъчно място за завършване на действието."
     IDS_ERROR_NOFONT "%1 не е редовен шрифтов файл."
     IDS_ERROR_NOCLASS "Неуспешно изпълнение на класа на прозореца."
+    IDS_ERROR_ISINSTALLED  "This font is already installed!" 
+    IDS_ERROR_FONTCPY "Failed to copy the font file!"
+    IDS_ERROR_OPENKEY "Failed to open the fonts key!"
+    IDS_ERROR_REGISTER "Failed to register the new font!"
+    IDS_SUCCESS "Success"
+    IDS_COMPLETED "Font Installation Completed."
```

Comment:
Not sure about the capitalization.
```suggestion
    IDS_COMPLETED "Font installation completed."
```

Reply:
fix: change capitalization, also for all language files